--- rucio/core/rse_counter.py	2019-07-12 10:22:15.881029868 -0500
+++ hack_rse_counter.py	2019-07-11 13:36:49.977219320 -0500
@@ -112,9 +112,9 @@
                           bindparam('total_workers', total_workers)]
             query = query.filter(text('ORA_HASH(rse_id, :total_workers) = :worker_number', bindparams=bindparams))
         elif session.bind.dialect.name == 'mysql':
-            query = query.filter('mod(md5(rse_id), %s) = %s' % (total_workers + 1, worker_number))
+            query = query.filter(text('mod(md5(rse_id), %s) = %s' % (total_workers + 1, worker_number)))
         elif session.bind.dialect.name == 'postgresql':
-            query = query.filter('mod(abs((\'x\'||md5(rse_id::text))::bit(32)::int), %s) = %s' % (total_workers + 1, worker_number))
+            query = query.filter(text('mod(abs((\'x\'||md5(rse_id::text))::bit(32)::int), %s) = %s' % (total_workers + 1, worker_number)))
 
     results = query.all()
     return [result.rse_id for result in results]
